include/core/parameters.h

```diff
@@ -456,6 +455,44 @@ namespace Parameters
     parse_parameters(ParameterHandler &prm);
   };
 
+  /**
+   * @brief Laser parameters - Defines the parameters for the
+   * laser heat source.
+   */
+  template <int dim>
+  class Laser
+  {
+  public:
+    // Laser parameters
+    bool   activate_laser;
+    double concentration_factor;
+    double laser_power;
+    double laser_absorptivity;
+    double penentration_depth;
+    double beam_radius;
+
+    enum class BeamDirection
+    {
+      x,
+      y,
+      z,
+    } beam_direction;
+
+    unsigned int beam_direction_coordinate;
+    unsigned int perpendicular_plane_coordinate_one;
+    unsigned int perpendicular_plane_coordinate_two;
```

Comment:
I'm not sure I understood what are perpendicular place coordinates. Are they two points in the plane? Also, where does this place originate?

Reply:
These auxiliary variables are used to store the coordinates of a perpendicular plane to the direction of emission. For instance, if the laser direction of emission is in the z coordinates (2), then perpendicular_plane_coordinate_one will be equal to 0 [x direction] and perpendicular_plane_coordinate_two will be equal to 1 [y direction]. I have added more descriptions to make these variables more clear.